server/src/main/java/org/opensearch/cluster/ClusterState.java

```diff
@@ -839,6 +839,34 @@ private static class ClusterStateDiff implements Diff<ClusterState> {
             minimumClusterManagerNodesOnPublishingClusterManager = after.minimumClusterManagerNodesOnPublishingClusterManager;
         }
 
+        @Override
+        public String toString() {
+            return "ClusterStateDiff{"
```

Comment:
Use StringBuilder?

Reply:
This was auto-generated. Will update anyways.